src/common/helpers/proposalTitleAndDescription/approvedIdentifiersTableParser.ts

```diff
@@ -0,0 +1,30 @@
+import fs from "fs";
```

Comment:
A simple script for parsing the approved identifiers data found in https://raw.githubusercontent.com/UMAprotocol/docs/master/docs/uma-tokenholders/approved-price-identifiers.md

We will automate this process for UMA 2.0